tests/backend_test.py

```diff
@@ -357,16 +357,14 @@ def test_quantinuum_sim_h11e_cond_2(azure_backend: AzureBackend) -> None:
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize("azure_backend", ["quantinuum.sim.h1-1e"], indirect=True)
 def test_quantinuum_option_params(azure_backend: AzureBackend) -> None:
-    c = Circuit(2, 2).H(0).CX(0, 1).measure_all()
+    c = Circuit(3, 3).H(0).CX(0, 1).H(2).measure_all()
     a_b = azure_backend
     c1 = a_b.get_compiled_circuit(c)
-    if a_b.is_available() and a_b.average_queue_time_s() < 600:
+    if a_b.is_available():
         h = a_b.process_circuit(c1, n_shots=1000, option_params={"error_model": False})  # type: ignore
         r = a_b.get_result(h, timeout=1200)
         counts = r.get_counts()
         assert all(x[0] == x[1] for x in counts)
         assert any(x[0] == 1 for x in counts)  # might fail in very rare cases
     else:
-        warnings.warn(
-            "quantinuum.sim.h1-1e unavailable or queue time >= 600s: not submitting"
-        )
+        warnings.warn("quantinuum.sim.h1-1e unavailable")
```

Comment:
Was this meant as a temporary change and should be reverted?

Reply:
This was intended to be a permanent change. The queue time is usually higher then 600s, but the actual queue time is much lower. And as this is the only check we are running with meaningful results it would be good to run this a bit more often to see issues early.